### PR TITLE
Modify model|add teacher model crud|fix some error✌

### PR DESCRIPTION
**Pull request by RatonBiswas**

### CODE REVIEW

File: contollers/sessionController.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import Session from '../model/sessionModel.js';
 import * as factory from './handlerFactory.js';
 // const factory = require('./handlerFactory');
 
-export const getAllCoursePost = factory.getAll(Session);
+export const getAllSessionPost = factory.getAll(Session);
 
-export const createCoursePost = factory.createOne(Session);
```
Comment (RatonBiswas) [inline contollers/sessionController.js:11]: fix some error😥

File: model/courseModel.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
 import mongoose from 'mongoose';
 
 const courseSchema = new mongoose.Schema({
-  course_name: {
```
Comment (RatonBiswas) [inline model/courseModel.js:4]: use Camel case

File: model/sessionModel.js [hunk 3]
```diff
@@ -13,20 +13,11 @@ const sessionSchema = new mongoose.Schema({
         type: String,
         required: [true, "A course allocation has a batch"]
     },
-    course: {
```
Comment (RatonBiswas) [inline model/sessionModel.js:16]: modify database design

File: model/sessionModel.js [hunk 4]
```diff
@@ -13,20 +13,11 @@ const sessionSchema = new mongoose.Schema({
...
-        course_code: {
-            type: String,
-            required: [true, 'A course must have a course code'],
-        },
-        credit: Number,
-        type: Object,
-        ref: 'Course',
-        required: [true, 'sessions must belong to Course.'],
-    }
+    sections:[String],
```
Comment (RatonBiswas) [inline model/sessionModel.js:16]: modify database model and add new field👏

File: model/teacherModel.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose';
```
Comment (RatonBiswas) [inline model/teacherModel.js:1]: Add teacher model👌

Comment (lgope) [reply model/teacherModel.js:1]: You did well. 👍

**Pull request merged**